.spelling

```diff
@@ -93,3 +93,6 @@ SystemLink
 webvi-examples
 USGS
 DataQueue
+LabVIEWExamples.lvproj
```

Comment:
We have been marking filenames / variable names, etc as code pieces and the spellcheck will avoid spelling errors on code pieces. ie wrap them in backticks like
```
`LabVIEWExamples.lvproj`
```

You can see that convention used for other paths and filenames in the READMEs

Reply:
Ah, that's the trick - done.